File: Pattern1/pattern01.java
//importing scanner class to take user input
import java.util.Scanner;

//main class
public class Main
{   
public static void pattern01(int n){ //function to print the result
    int i,j;   //variables for rows and columns
    for(i=0;i<n;i++){  //row operation
        for(j=0;j<n;j++){  //column operation
            System.out.print("1 ");  //printing in the format
        }
        System.out.println(); //moving the cursor to next line
    }
}

	public static void main(String[] args) { // main method
	    Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();  //taking input from user for the o/p
		pattern01(n);
	}
}

File: Pattern1/pattern01.cpp
// including header files
#include <iostream>
using namespace std;

//function to print the pattern
void pattern01(int n)
{
    int i,j; // variables for rows and columns
    for(int i=0;i<n;i++){ //row operation
        for(j=0;j<n;j++){ // column operation
            cout<<"1 ";   // printing the desired o/p
        }
        cout<<endl; //changing thepointer to next line i.e moving to next row
    }
}

// main function 
int main()
{
    int n;
    pattern01(n); // calling the function to print the output
    return 0;
}
